docs/docs/guides/eth-network-dapp.md

```diff
@@ -0,0 +1,350 @@
+                                                              validator-metrics: 5064/tcp -> 127.0.0.1:65029
+c34417bea5fa   cl-client-2                                    http: 4000/tcp -> 127.0.0.1:65037                RUNNING
+                                                              metrics: 8008/tcp -> 127.0.0.1:65035
+                                                              tcp-discovery: 9000/tcp -> 127.0.0.1:65036
+                                                              udp-discovery: 9000/udp -> 127.0.0.1:63581
+e19738e6329d   el-client-0                                    engine-rpc: 8551/tcp -> 127.0.0.1:64986          RUNNING
+                                                              rpc: 8545/tcp -> 127.0.0.1:64988
+                                                              tcp-discovery: 30303/tcp -> 127.0.0.1:64987
+                                                              udp-discovery: 30303/udp -> 127.0.0.1:55706
+                                                              ws: 8546/tcp -> 127.0.0.1:64989
+e904687449d9   el-client-1                                    engine-rpc: 8551/tcp -> 127.0.0.1:64993          RUNNING
+                                                              rpc: 8545/tcp -> 127.0.0.1:64995
+                                                              tcp-discovery: 30303/tcp -> 127.0.0.1:64994
+                                                              udp-discovery: 30303/udp -> 127.0.0.1:58096
+                                                              ws: 8546/tcp -> 127.0.0.1:64996
+ad6f401126fa   el-client-2                                    engine-rpc: 8551/tcp -> 127.0.0.1:65003          RUNNING
+                                                              rpc: 8545/tcp -> 127.0.0.1:65001
+                                                              tcp-discovery: 30303/tcp -> 127.0.0.1:65000
+                                                              udp-discovery: 30303/udp -> 127.0.0.1:57269
+                                                              ws: 8546/tcp -> 127.0.0.1:65002
+12d04a9dbb69   prelaunch-data-generator-1680882122181135513   <none>                                           STOPPED
+5b45f9c0504b   prelaunch-data-generator-1680882122192182847   <none>                                           STOPPED
+3d4aaa75e218   prelaunch-data-generator-1680882122201668972   <none>                                           STOPPED
+```
+Congratulations! You’ve successfully configured your local testnet to have 2 nodes instead of 1. To run the same workflows you did before against your dApp (deploy & test), perform the same operations we did before by replacing the `&lt;PORT>` in the `localnet` struct in your `hardhat.config.ts config file `with the port of the rpc uri output from any `el-client-&lt;num>`  service in your new, 2-node local testnet.
```

Comment:
```suggestion
Congratulations! You’ve successfully configured your local testnet to have 2 nodes instead of 1. To run the same workflows you did before against your dApp (deploy & test), perform the same operations we did before by replacing the `<PORT>` in the `localnet` struct in your `hardhat.config.ts` config file with the port of the rpc uri output from any `el-client-<num>`  service in your new, 2-node local testnet.
```